### PR TITLE
Use oc 3.11; set default routing

### PR DESCRIPTION
**Pull request by jpechane**

### CODE REVIEW

**Comment by gunnarmorling** [comment]
@jpechane Did you observe any regression in terms of start-up time? 3.10 was much slower than 3.9 for me. Not sure how it's with 3.11?

**Comment by jpechane** [comment]
The start is definitely slower but IMHO it is because more componenets are started

File: vm-setup/site.yml [hunk 1]
```diff
@@ -90,13 +89,3 @@
 
     become: true
     become_user: root
-  - name: configure OpenShift
-    block:
-      - name: Create OpenShift config files
-        shell: "env PATH={{ tools_dir }}/openshift-origin-client-tools-v{{ openshift_version }}-{{ openshift_hash }}-linux-64bit:$PATH bash -c 'oc cluster up && oc cluster down'"
-      - name: Setup OpenShift config files
```
Comment (gunnarmorling) [inline vm-setup/site.yml]: @jpechane Just seeing this. Why did it get removed?

File: voxxed-microservices-2018/module-01.adoc [hunk 2]
```diff
@@ -490,3 +490,11 @@ This time the output should be empty.
 
 Now your Kafka cluster is running and ready to go.
 Let's build some applications!
+
+==== Environment cleanup
+
+After you complete all modules in this exercise then you can clean up the environment by issuing commands
+```console
+$ oc cluster down
+$ rm $HOME/openshift.local.clusterup
```
Comment (gunnarmorling) [inline voxxed-microservices-2018/module-01.adoc]: s/rm/rm -rf/

File: voxxed-microservices-2018/module-01.adoc [hunk 3]
```diff
@@ -490,3 +490,11 @@ This time the output should be empty.
 
 Now your Kafka cluster is running and ready to go.
 Let's build some applications!
+
+==== Environment cleanup
+
+After you complete all modules in this exercise then you can clean up the environment by issuing commands
+```console
```
Comment (gunnarmorling) [inline voxxed-microservices-2018/module-01.adoc]: Can we stick to the `[source, sh]` style? Same below.

File: vm-setup/site.yml (hunk 1 above)
Comment (jpechane) [reply vm-setup/site.yml]: Beacuse the pod limit is 250 in 3.11

File: voxxed-microservices-2018/module-01.adoc (hunk 2 above)
Comment (gunnarmorling) [reply voxxed-microservices-2018/module-01.adoc]: Hum, I'm getting

rm: cannot remove '../openshift.local.clusterup/etcd/member': Permission denied
rm: cannot remove '../openshift.local.clusterup/openshift.local.volumes/pods': Permission denied

**Pull request merged**